salt/_modules/caasp_net.py

```diff
@@ -6,6 +6,13 @@
 from __future__ import absolute_import
 
 
+# note: do not import caasp modules other than caasp_log
+from caasp_log import error
+
+
+DEFAULT_INTERFACE = 'eth0'
```

Comment:
Maybe it would be better to set that via a pillar

Reply:
Ok, I'll do that